backend/seeds/seeds_test.go

```diff
@@ -26,7 +26,7 @@ func TestSeedInitialDataWithContext(t *testing.T) {
 	s := new(storetest.Store)
 	s.On("NamespaceStore").Return(nsStore)
 	s.On("CreateIfNotExists", mock.Anything, mock.Anything, mock.Anything).Return(nil)
-	s.On("Initialize", mock.Anything, mock.Anything).Return(seedCluster(ctx, s, nsStore, config)(ctx))
```

Comment:
Looks like this test is failing. Needs to import sensu-go/backend/store and include this mock: `nsStore.On("Get", mock.Anything, mock.Anything).Return(nil, new(store.ErrNotFound))`

Reply:
Semi unrelated note. May be worth while moving away from store.ErrNotFound type.